modified_cdc_protocol/operation_types/rehydratepatientsample/protocol.rb

```diff
@@ -0,0 +1,171 @@
+# typed: false
+# frozen_string_literal: true
+
+needs 'Standard Libs/PlanParams'
+needs 'Standard Libs/Debug'
+needs 'Standard Libs/Pipettors'
+needs 'Standard Libs/Units'
+needs 'Standard Libs/AssociationManagement'
+
+needs 'Collection Management/CollectionDisplay'
+needs 'Collection Management/CollectionTransfer'
+
+needs 'Diagnostic RT-qPCR/DiagnosticRTqPCRHelper'
+
+needs 'Tube Rack/TubeRack'
+
+needs 'Modified CDC Protocol/RehydratePatientSampleDebug'
+
+TEMPLATE = 'Template'
+MEDIA = 'Media'
+
+EPP_LABEL = 'Eppendorf Tube'
+
+class Protocol
+  # Standard Libs
+  include PlanParams
+  include Debug
+  include Pipettors
+  include Units
+  include AssociationManagement
+  include PartProvenance
+
+  #Collection Management
+  include CollectionDisplay
+  include CollectionTransfer
+
+  #Diagnostic RT-qPCR
+  include DiagnosticRTqPCRHelper
+
+  #TubeRack
+  include TubeRackHelper
+
+  #Modified CDC Protocol
+  include RehydratePatientSampleDebug
+
+  attr_reader :job_params, :operation_params, :media_volume
+  def default_operation_params
+    {
+      media_volume: 300,
+      media_units: MICROLITERS,
+    }
+  end
+
+  def default_job_params
+    {
+    }
+  end
+
+  def main
+    setup_test(operations)
+    @job_params = update_all_params(
+      operations: operations,
+      default_job_params: default_job_params,
+      default_operation_params: default_operation_params
+    )
+
+    rnase_warning
+
+    safety_warning
+
+    operations.retrieve.make
+
+    operations.each do |op|
+      operation_params = op.temporary[:options]
+      media_volume = {
+        qty: operation_params[:media_volume],
+        units: operation_params[:media_units]
+      }
+
+      op.pass(TEMPLATE)
+
+      item_list = op.input_array(TEMPLATE).map do |fv|
+        fv.item.collection? ? fv.collection : fv.item
+      end
+
+      if item_list.first.collection?
+        item_list.uniq
+        sample_rack = StripWellRack.new(rack_type: '96 Well Strip Well Tube Rack')
+      else
+        sample_rack = TubeRack.new(rack_type: '96 Well Strip Well Tube Rack')
+      end
+
+      fetch_tube_rack(sample_rack)
+
+      add_items(item_list, sample_rack)
+
+      transfer_media_to_collection(sample_rack: sample_rack,
+                                   media: op.input(MEDIA).item,
+                                   volume: media_volume,
+                                   rc_list: sample_rack.collection.get_non_empty)
+      vortex_samples(sample_rack: sample_rack)
+    end
+    operations.store
+
+    {}
+  end
+    
+  # Directions to vortex samples
+  # @aparam sample_rack [SampleRack]
+  # @param rc_list [Array<[r,c]>] list of all locations that need media
+  def vortex_samples(sample_rack:, rc_list: nil)
```

Comment:
Prefix method names that are primarily show blocks with `show_` and put them all together.

Reply:
Yes, this is a new rule I came up with 😄